src/Microsoft.OData.Client/Binding/BindingObserver.cs

```diff
@@ -427,8 +427,8 @@ internal void OnPrimitiveOrComplexCollectionChanged(object sender, NotifyCollect
                     case NotifyCollectionChangedAction.Reset:
                         this.bindingGraph.RemoveCollection(sender);
                         break;
-                    default:
-                        throw new InvalidOperationException(Strings.DataBinding_CollectionChangedUnknownActionCollection(e.Action, sender.GetType()));
+                    default: // No Action

```

Comment:
Ditto. Same comment as above.

Reply:
As Above, I think this is okay.

---
In reply to: [355600737](https://github.com/OData/odata.net/pull/1601#discussion_r355600737) [](ancestors = 355600737)